src/components/contact-info-section/ContactInfoSection.tsx

```diff
@@ -61,7 +61,7 @@ const ContactInfoSection: FC<ContactInfoSectionProps> = ({
         required
         onChange={(newEmail) => changePatron(newEmail, "emailAddress")}
         value={patron?.emailAddress}
-        label={t("patronContactEmailLabelText")}
+        label={`${t("patronContactEmailLabelText")}*`}
```

Comment:
I think it should be the responsibility of the `TextInput` component to prefix label text with an asterisk if it is required.

Reply:
Good point